frontend/public/components/markdown-view.tsx

```diff
@@ -186,9 +189,15 @@ const IFrameMarkdownView: React.FC<InnerSyncMarkdownProps> = ({
   );
 
   const onLoad = React.useCallback(() => {
+    const htmlTagElement = frame?.contentDocument?.documentElement;
+    if (themeLoaded) {
+      updateThemeClass(htmlTagElement, theme);
+    } else {
+      updateThemeClass(htmlTagElement, localTheme);
+    }
```

Comment:
Just a question: Is it possible to add the theme to the generated HTML (contents string below)?

What happens when the theme changes in another browser tab? Do we need a useEffect here as well to call `updateThemeClass`?